testing/dart/observatory/skp_test.dart

```diff
@@ -10,31 +10,22 @@ import 'dart:developer' as developer;
 import 'dart:typed_data';
 import 'dart:ui';
 
-import 'package:test/test.dart';
+import 'package:litetest/litetest.dart';
 import 'package:vm_service/vm_service.dart' as vms;
 import 'package:vm_service/vm_service_io.dart';
 
 void main() {
-  late vms.VmService vmService;
-
-  setUpAll(() async {
```

Comment:
Replacing `setUpAll` and `tearDownAll` by inlining the code directly into the single test.